modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/PortletExportControllerImpl.java

```diff
@@ -113,8 +112,11 @@
  * @author Máté Thurzó
  */
 @Component(
-	property = "model.class.name=com.liferay.portal.kernel.model.Portlet",
-	service = {ExportImportController.class, PortletExportController.class}
+	property = {
+		"export.controller=true",
```

Comment:
Hey @GabrielHenriquedev, the change in the tracker that supports this modification should also be in this commit. Thx